### PR TITLE
Plan Details on upgrade page

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 

### Preview
screen shot 2016-09-23 at 2 59 55 pm

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 30e1abb6f666...1568f62ab753 introduced some visual changes on linux:
 * Added: **Plan_Details-Credit_Card_With_Past**  
   Plan_Details-Credit_Card_With_Past rendered
 * Added: **Plan_Details-Credit_Card_No_Past**  
   Plan_Details-Credit_Card_No_Past rendered
 * Added: **Plan_Details-Apple_Pay**  
   Plan_Details-Apple_Pay rendered
 * Changed: **Landing-Normal_Not_Verified_email** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474668443002-30e1abb6f666-1568f62ab753-linux/Landing-Normal_Not_Verified_email-30e1abb6f666.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474668443002-30e1abb6f666-1568f62ab753-linux/Landing-Normal_Not_Verified_email-1568f62ab753.png)  
   Landing-Normal_Not_Verified_email comparison
 * Changed: **Landing-Normal** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474668443002-30e1abb6f666-1568f62ab753-linux/Landing-Normal-30e1abb6f666.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474668443002-30e1abb6f666-1568f62ab753-linux/Landing-Normal-1568f62ab753.png)  
   Landing-Normal comparison
 * Changed: **Landing-Gold_Plan_Broken_Payment** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474668443002-30e1abb6f666-1568f62ab753-linux/Landing-Gold_Plan_Broken_Payment-30e1abb6f666.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474668443002-30e1abb6f666-1568f62ab753-linux/Landing-Gold_Plan_Broken_Payment-1568f62ab753.png)  
   Landing-Gold_Plan_Broken_Payment comparison
 * Changed: **Landing-Gold_Plan** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474668443002-30e1abb6f666-1568f62ab753-linux/Landing-Gold_Plan-30e1abb6f666.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474668443002-30e1abb6f666-1568f62ab753-linux/Landing-Gold_Plan-1568f62ab753.png)  
   Landing-Gold_Plan comparison

File: shared/settings/common.desktop.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <Box style={globalStyles.flexBoxRow}>
+      {range(starCount).map(i => <Icon key={i} style={{color: globalColors.green}} type='iconfont-star' />)}
+    </Box>
+  )
+}
+
+export {
+  Stars,
+}
+
```
Comment (chromakode) [inline shared/settings/common.desktop.js]: :scissors: extra newline

File: shared/constants/settings.js [hunk 2]
```diff
@@ -1,11 +1,63 @@
...
+}
+
+const levelToSpace: {[key: PlanLevel]: string} = {
+  'Basic': '10GB',
+  'Gold': '50GB',
+  'Friend': '250GB',
+}
+
+function levelToDetails (p: PlanLevel) {
+  return `You will be able to use up to ${levelToSpace[p]} of data`
```
Comment (chromakode) [inline shared/constants/settings.js]: I think you should add a period to the end of the sentence here to match the mockups.

File: shared/settings/common.desktop.js (hunk 1 above)
Comment (chromakode) [inline shared/settings/common.desktop.js]: :scissors: Extra newline here!

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 30e1abb6f666...1a1fc9476e2f introduced some visual changes on linux:
 * Added: **Plan_Details-Credit_Card_With_Past**  
   Plan_Details-Credit_Card_With_Past rendered
 * Added: **Plan_Details-Credit_Card_No_Past**  
   Plan_Details-Credit_Card_No_Past rendered
 * Added: **Plan_Details-Apple_Pay**  
   Plan_Details-Apple_Pay rendered
 * Changed: **Landing-Normal_Not_Verified_email** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474931440012-30e1abb6f666-1a1fc9476e2f-linux/Landing-Normal_Not_Verified_email-30e1abb6f666.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474931440012-30e1abb6f666-1a1fc9476e2f-linux/Landing-Normal_Not_Verified_email-1a1fc9476e2f.png)  
   Landing-Normal_Not_Verified_email comparison
 * Changed: **Landing-Normal** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474931440012-30e1abb6f666-1a1fc9476e2f-linux/Landing-Normal-30e1abb6f666.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474931440012-30e1abb6f666-1a1fc9476e2f-linux/Landing-Normal-1a1fc9476e2f.png)  
   Landing-Normal comparison
 * Changed: **Landing-Gold_Plan_Broken_Payment** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474931440012-30e1abb6f666-1a1fc9476e2f-linux/Landing-Gold_Plan_Broken_Payment-30e1abb6f666.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474931440012-30e1abb6f666-1a1fc9476e2f-linux/Landing-Gold_Plan_Broken_Payment-1a1fc9476e2f.png)  
   Landing-Gold_Plan_Broken_Payment comparison
 * Changed: **Landing-Gold_Plan** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474931440012-30e1abb6f666-1a1fc9476e2f-linux/Landing-Gold_Plan-30e1abb6f666.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474931440012-30e1abb6f666-1a1fc9476e2f-linux/Landing-Gold_Plan-1a1fc9476e2f.png)  
   Landing-Gold_Plan comparison

**Pull request merged**